apps/server/src/services/report-service/ReportService.ts

```
@@ -0,0 +1,62 @@
+export function clear(id?: string) {
+  if (id) {
+    report.delete(id);
+  } else {
+    report.clear();
+  }
+}
+
+export function eventStart(state: RuntimeState) {
+  if (state.eventNow === null) {
+    // eslint-disable-next-line no-unused-labels -- dev code path
+    DEV: {
```

Biome

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)